R/direct_estimation.R

```diff
@@ -48,8 +48,7 @@
 #' @param custom_indicator a list of functions containing the indicators to be
 #' calculated additionally. Such functions must and must only depend on the
```

Comment:
If I understand correctly that the functions do not need to be dependent on the threshold and pop_weights, I think we should adjust the documentation to: Such functions must only depend on the target ....